Jenkinsfile

```diff
@@ -799,9 +294,9 @@ pipeline {
                         }
                         */
                         always {
-                            sh '''mv test_results/*.xml run_test.sh/'''
-			    junit 'test_results/*.xml'
-                            archiveArtifacts artifacts: 'run_test.sh/**'
+				sh '''mv test_results/ run_test.sh/'''
```

Comment:
This is going to put the `test_results/` dir into the `run_test.sh/` dir which will make your `junit` below look in the wrong place for the junit files.

Reply:
> I don't think this is the fix for your previous problem.
> 
> This is going to put the `test_results/` dir into the `run_test.sh/` dir which will make your `junit` below look in the wrong place for the junit files.
> 
> You probably need to back up and make sure that there are `.xml` files in `test_results/` before you go any further. Put an `ls -l test_results/` right at the top of this `sh` script to see what's in there when the tests are done.
 ok. let me add the ls -l test_results.